### DISCUSSION TITLE
Calificaciones. Ejercitacion #1. S3P2

### DISCUSSION THREAD

**Discussion by gonaiot** [ISPC-TST-EM2022/Grupo1#67] [Announcements]
Calificacion Grupal: 7(siete)

Calificaciones Personales. 
@FedeBirge : 10 (diez)
@Emygut : 6 (seis) 
@marlufer : 6 (seis)
@GiseRomero : 6 (seis)
@LitoCarunchio : 6 (seis)

Observaciones: Estimados, trabajaron muy bien, estuvieron soportados por el buen trabajo de @FedeBirge. Aun asi sean concientes de que la conformacion de un equipo de alto desempeño requiere el esfuerzo y la colaboracion de todos en forma equitativa. Permitanse asignar roles diferentes en los proximos tp, y vean que sucede. Saludos coordiales.

**Comment by GiseRomero** [comment]
Muchas gracias profe por la devolución, lo tendremos en cuenta!

**Comment by Emygut** [comment]
Gracias Profesor por las devoluciones y observaciones se tendra en cuenta en los proximos TP.

**Comment by marlufer** [comment]
Perfecto Profe. Tendremos en cuenta las marcaciones.

Saludos
